app/src/main/java/org/wikipedia/analytics/metricsplatform/MetricsPlatform.kt

```diff
@@ -19,6 +20,7 @@ object MetricsPlatform {
         "WikipediaApp/" + BuildConfig.VERSION_NAME,
         "android",
         "app",
+        Build.MODEL + Build.DEVICE,
```

Comment:
@dbrant is this the right data to populate `device_family` in the agent data object?

Reply:
@cjming It depends -- I don't think I'm familiar with the purpose of the `device_family` field; What would we like it to be?

Reply:
Ah sorry, I see a [comment](https://phabricator.wikimedia.org/T357371#9746840) that basically requests the `device_family` to be the same as what we're currently sending in our User-Agent.

If we want it to be exactly the same as the current User-Agent, then it should be just `Build.MODEL`, but if we need it to be like Shay requests in the above comment, it should be `Build.BRAND + " " + Build.MODEL`.

Reply:
thanks! that's perfect - updated accordingly